README.md

```diff
@@ -98,27 +131,30 @@ More information can be found in Hashicorp's [Production Hardening](https://lear
 After the initial configuration, it is a good idea to minimize the risk of exposure by revoking the root token.
 
 ```bash
-curl --request POST http://${VAULT_IP_ADDR}:8200/v1/auth/token/revoke-self --header "X-Vault-Token: ${ROOT_TOKEN}"
+curl --request POST http://${DOMAIN_NAME}:8200/v1/auth/token/revoke-self --header "X-Vault-Token: ${ROOT_TOKEN}"
 ```
 
 If needed, the root token can be regenerated, but this requires the recovery keys. Steps can be followed using the [CLI](https://learn.hashicorp.com/tutorials/vault/generate-root) or [API](https://www.vaultproject.io/api-docs/system/generate-root).
 
 #### Disabling SSH Access
 
-The deployment left the instance's SSH ports open for convenience during deployment. They are a frequent point of attack and should be disabled in Azure Portal.
+The deployment left all VMs' SSH ports open for convenience during deployment. They are a frequent point of attack and should be disabled in Azure Portal.
 
-When everything works correctly, the VM scale set `vault-vmss`, in the Networking section, has an inbound port rule named `ssh` that can be deleted. It can be created anew when needed.
+When everything works correctly, both `vault-nsg` and `nginx-nsg` have inbound security rules named `ssh` that can be deleted. They can be created anew when needed.
 
 ### Debugging
 
-If something goes wrong, SSH access to one of the VM might be necessary. Unless configured otherwise, this can only be done from where Terraform was run, since this is where the SSH public key was copied to the VM scale set `vault-vmss`.
+If something goes wrong, SSH access to one of the VM might be necessary. Unless configured otherwise, this can only be done from where Terraform was run, since this is where the SSH public key was copied.
```

Comment:
When you run terraform, you can give any public key, so it's not really true.
And, you can reset the ssh key if needed with Azure

Reply:
That's what I meant with "Unless configured otherwise". By default, it will take the ssh key on the current machine.

Reply:
I simply removed the `Unless [...]`. You're right that the reader is supposed to know how to get around Terraform and Azure.